packages/effect/src/internal/schema/fromJsonSchemaDocument.ts

```
@@ -0,0 +1,981 @@
+              {
+                _tag: "Literal",
+                literal: right.literal,
+                checks: right.checks
+              },
+              left,
+              right
+            )
+            : { _tag: "Never", checks: [] }
+        }
+        if (right._tag !== "String") return { _tag: "Never", checks: [] }
+        const stringChecks = combineChecks(left.checks, right.checks, right.annotations)
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)